### PR TITLE
Feat/comment component

### PR DESCRIPTION
**Pull request by aseiden**
tl;dr Adds a comments page component, comments banner component, and a comments chat component.  All together these will provide the scaffolding and some of the content for our comments page.

CommentsBanner.jsx: Pretty much a rip of results list item right now, but made separate in case we want to change it up a bit.  Can condense back into one component if we don't end up styling this one any differently.

CommentsChat.jsx: Stub for the comments chat box we'll be making.

CommentsPage.jsx: Holding component for CommentsBanner and CommentsChat.  Also manages talking to the server (though perhaps CommentsChat will need to talk as well, depending on how the socket stuff works)

controllers.js: Added a college route for getting a single college from the db based on a specific id.

models.js: Added a college query.

router.js: Added a college route.

### CODE REVIEW

File: client/src/components/CommentsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      university: {}
+    };
+  }
+
+  componentDidMount() {
+    this.getBanner(1);
```
Comment (aseiden) [inline client/src/components/CommentsPage.jsx:18]: 1 is a placeholder, will add the react router route param once we have that implemented.

**Review by epan — APPROVED** [review]
kewl beenz
